### ISSUE TITLE
The tab controls produced by the package are not accessible to screen readers

### ISSUE THREAD

**Issue by gtritchie** [rstudio/shinydashboard#417]
To be usable via screen readers, Tab controls need to follow this pattern: https://www.w3.org/WAI/ARIA/apg/patterns/tabs/.

Currently they do not; a temporary workaround I concocted can be seen in https://github.com/rstudio/shinydashboarda11y (not an official solution, just something I threw together and only tested very briefly).
